consensus/enclave/impl/src/lib.rs

```diff
@@ -50,7 +50,7 @@ use mc_sgx_report_cache_api::{ReportableEnclave, Result as ReportableEnclaveResu
 use mc_transaction_core::{
     constants::MINIMUM_FEE,
     membership_proofs::compute_implied_merkle_root,
-    onetime_keys::{create_onetime_public_key, create_shared_secret, create_tx_public_key},
+    onetime_keys::{create_shared_secret, create_tx_public_key, create_tx_target_key},
```

Comment:
Suggestion: maybe these should be 'create_output_public_key' and 'create_output_target_key', or 'create_tx_out_public_key' and 'create_tx_out_target_key'. I think the function name 'create_tx_public_key' hearkens back to when a transaction had a single public key that was shared by every output. Now, each output has a distinct target key and public key.